primitives/src/sequence.rs

```diff
@@ -205,6 +205,14 @@ impl Sequence {
     fn low_u16(&self) -> u16 { self.0 as u16 }
 }
 
+#[cfg(feature = "arbitrary")]
```

Comment:
We are trying to group these all at the bottom of the file directly above the tests if that makes sense.

Reply:
I don't see any of them directly above the tests. They're grouped with the other `impl`s which makes more sense to me

Reply:
See https://news.ycombinator.com/item?id=41544364.  Personally I think your argument to group with impls makes sense.  Ideally wherever they are placed it's consistent.